pkg/api/folder.go

```diff
@@ -31,7 +31,7 @@ const REDACTED = "redacted"
 //
 // Get all folders.
 //
-// Returns all folders that the authenticated user has permission to view.
+// It returns all folders that the authenticated user has permission to view.
```

Comment:
this change is only for triggering the `swagger-gen` step (triggered only if there are backend changes)